### PR TITLE
Added EMAIL_ADDRESS test case

### LINKED ISSUE #4: Detect EMAIL_ADDRESS
Description:
An email address is a type of PII that can directly reveal personal details such as identity, contact information, or even location of the individual. 

Detection Steps:
An Email Address basically follows the standard format of having a username (which could include letters, numbers, and special characters). Then the @ symbol which separates the username and domain. Lastly would be the domain, which also contains letters, numbers and special characters followed by .com, .org, and the list goes on.

### PR DESCRIPTION
**Pull request by skybot00**
Added email address test case

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
 	"name": "Python 3",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/python:0-3.11",
+	"image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
```
Comment (siblount) [inline .devcontainer/devcontainer.json]: Please revert this back so it doesn't apply to everyone else.

File: test_team_new_orleans_fc.py [hunk 2]
```diff
@@ -20,7 +20,36 @@ def test_date_time(self):
 
     def test_email_address(self):
```
Comment (claesmk) [inline test_team_new_orleans_fc.py]: You can remove test_email_address() since you have broken out positive and negative test cases

Comment (skybot00) [reply test_team_new_orleans_fc.py]: Okay, I will do that.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (skybot00) [reply .devcontainer/devcontainer.json]: yup, thats my bad. I fixed it.

**Review by Camfrm336 — APPROVED** [review]
Looks like you have all the different types of email formats, looks great!

**Review by claesmk — APPROVED** [review]
test_email_address_negatives() was over-indented so it was not actually running.  I corrected that.  Looks OK now.

**Pull request merged**